src/redist/targets/packaging/windows/clisdk/dotnet.wxs

```diff
@@ -59,4 +60,11 @@
       </Directory>
     </Directory>
   </Fragment>
+  <Fragment>
+    <ComponentGroup Id="DotnetToolSetup">
+      <Component Id="AppendDotnetToolShimPath" Directory="TARGETDIR" Guid="{241B7B8D-7817-41C1-807F-5F4010BF583A}">
+        <Environment Id="E_PATH" Name="PATH" Value="%USERPROFILE%\.dotnet\tools" Part="last" Action="set" System="no" />
```

Comment:
We tried doing this before with the old DNX installers. You can set machine wide paths, but doing user specific actions for a per-machine install can cause a lot of problems.

For example, the MSI could be run under a different account, e.g. think of an IT process doing installs on users machines because they don't have admin rights. The install would run under a different user. Or worse, it runs under the Trusted Installer service.

I'll forward a thread on this to you

Reply:
Directory="DOTNETHOME" Guid="*"

Reply:
Talked offline, it is `System="no"` this time. It won't work for all scenario as in if user A ran the installer, and user B won't have it. However, we have a fallback experience in the first run, so user B won't be blocked (but need to restart the command prompt)